resources/assets/js/videos/components/videoScreen.vue

```diff
@@ -452,10 +452,18 @@ export default {
         extractAnnotationFromFeature(feature) {
             return feature.get('annotation');
         },
-        handleFeatureSelect(e) {
+        handleFeatureSelect() {
+            // Don't use event.deselected here because it uses selections in current frame only,
+            // see https://github.com/biigle/core/issues/552
+
+            // New selected annotations
+            let selected = this.selectInteraction.getFeatures().getArray().map(this.extractAnnotationFromFeature);
```

Comment:
This should be equivalent to `e.selected.map(this.extractAnnotationFromFeature)`. Why not use it?

Reply:
If you use this, then you cannot select multiple annotations in one frame with shift any more.

Reply:
Ah I see, it's because `e.selected` contains only the newly selected annotations and any other selected annotations would be deselected by the new logic for `deselected` again.